cli/rt/27_websocket.js

```diff
@@ -201,7 +201,7 @@
     }
 
     close(code, reason) {
-      if (code && (code !== 1000 && !(3000 <= code > 5000))) {
+      if (code && code !== 1000 && !(code >= 3000 && code < 5000)) {
```

Comment:
Thanks for the PR - I'm not sure how the previous code even got through lint!

I think it would be better if this was written:
`if (code === 1000 || (3000 <= code && code < 5000))`

Reply:
sure. I editted